### PR TITLE
cilium-compose: Add IFACE env variable to configure uplink device

### PR DESCRIPTION
**Pull request by tgraf**
Reported and suggested by @salv-orlando

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -12,3 +12,4 @@ project:
 
 Alexei Starovoitov    ast@plumgrid.com
 Brenden Blanco        bblanco@plumgrid.com
+Salvatore Orlando
```
Comment (salv-orlando) [inline AUTHORS]: oh well, such a small think should not qualify me as an author.
anyway my email is salv.orlando@gmail.com

File: examples/docker-compose/README.md [hunk 2]
```diff
@@ -79,4 +83,4 @@ PING f00d::c0a8:2115:74ca(f00d::c0a8:2115:74ca) 56 data bytes
 ![Cilium dashboard](cilium-docker-4.png)
 
 If you try to ping from `client` directly to `wine` it won't be possible since the policy
-doesn't allow it.
\ No newline at end of file
+doesn't allow it.
```
Comment (salv-orlando) [inline examples/docker-compose/README.md]: can's spot the diff here...

**Comment by salv-orlando** [comment]
Thanks for filing this pull request.
This is what I was looking for.

File: examples/docker-compose/README.md (hunk 2 above)
Comment (tgraf) [reply examples/docker-compose/README.md]: My vim go plugin automatically fixes newlines at the end of a file.

**Comment by tgraf** [comment]
Thanks for the review

**Pull request merged**